### ISSUE TITLE
reloadCallback

### ISSUE THREAD

**Issue by sali1982** [Zizzamia/ng-tasty#181]
Hi,

Thanks for this great component.

I have an issue. `bind-reload="reloadCallback"` works fine when I have one table.

I use a method for dynamicly adding tables to the page. 
So I push an object with functions and attributes.

Everything is fine, except the reload one.

Example:
```html
<uib-accordion-group ng-repeat=" xx in xxes"  style="margin:5px;" is-open="true">
<div tasty-table bind-resource-callback="xx.getFiles" 
bind-list-items-per-page="xx.pageFilters" bind-init="hash.init" 
bind-theme="xx.myTheme" bind-filters="xx.filterBy" bind-reload="xx.reloadCallback" >
</div>
```

And xxes is: 
```js
 tableObj = {
  title: xx, 
  header: header, 
  init : init, 
  filterBy: filterBy, 
  filter : filter, 
  files : files, 
  getFiles : getFiles, 
  reloadCallback : reloadCallback
};
```
All of these are like:
```js
 var getFiles = function() {......}
```
They are pushed to xxes
```js
xxes.push(tableObj);
```
So everything is fine I have dynamic table but the reloadCallback is not working and when it is added, the whole table is not loading... 

Thanks

**Comment by Zizzamia** [comment]
I will make a documentation page to show how do it, and if does not work I will fix it :-)

**Comment by Zizzamia** [comment]
Here a demo that could help you understand how make your code
http://zizzamia.com/ng-tasty/directive/table-server-side/reload

This demo use `ngTasty v0.6.0` let me know if there is something is not working.
Thanks

**Comment by Zizzamia** [comment]
If there is nothing more about this issue I'm going to close it!
Thanks

**Comment by Zizzamia** [comment]
Happy new Year ✨ just released the latest version `v0.6.1` of ngTasty

**Comment by Disane87** [comment]
I have the same issue and studied your doc about this topic. If I set the attrib bind-reload="getResource" the entire table is not rendered:
![image](https://cloud.githubusercontent.com/assets/956158/17810992/aab3fc36-6620-11e6-8ba2-1ed90b7364d5.png)

If I remove the attribs, it woks pretty fine.

My table:

``` html
<div tasty-table bind-filters="filters" bind-resource-callback="getResource" bind-reload="getResource" bind-init="init">
                    <table class="table table-striped table-condensed">
                      <thead tasty-thead></thead>
                      <tbody>
                        <tr ng-repeat="row in rows">
                          <!-- <td>{{ row.ID }}</td> -->
                          <td>
                            <span action-button color="btn-success" itemid="row.ID" type="'recipe'"></span>
                          </td>
                          <td>{{ row.Name }}</td>
                          <td>{{ row.Basename }}</td>
                          <td>{{ row.Menge  | volumne }}</td>
                          <td><uib-progressbar type="{{row._TYPE}}" value="(row.Restmenge / row.Menge) * 100">{{ row.Restmenge  | volumne }}</uib-progressbar></td>
                          <td>{{ row.Nikotin }}mg</td>
                          <td>{{ row.Reifezeit }} {{ 'DAYS' | translate:translationData }}</td>
                          <td>{{ row.Preis | currency }}</td>
                          <td>{{ row.Herstellungsdatum | amUtc | amDateFormat:'L' }}</td>
                          <td>{{ row.Benutzer }}</td>
                          <!-- <td>{{ row.Bewertung }}</td> -->

                          <!-- <td>{{ row.Eigenschaften }}</td> -->

                        </tr>
                      </tbody>
                    </table>
                    <div tasty-pagination bind-list-items-per-page="listItemsPerPage"></div>
                  </div>
```

My controller:

``` js
function databaseRecipeCtrl($rootScope, $scope, Item, Data, Notify, $timeout) {
    // Notify.send("ghfhfhhhfhhf");

    // $scope.radioModel = 10;
    $scope.reloadCallback = function () {};
    $scope.init = {
      'count': 10,
      'page': 1,
      'sortBy': 'name',
      'sortOrder': 'dsc',
      'filterBase': 1 // set false to disable
    };
    $scope.listItemsPerPage = [10, 25, 50, 100];

    $scope.getResource = function (params, paramsObj) {

      return Data.post('recipes', paramsObj).then(function(results) {
        //
        return {
          'rows': results.rows,
          'header': results.header,
          'pagination': results.pagination[0]
        }
      });
    }
    $rootScope.$on('UpdateRecipeTable',   $scope.reloadCallback());
}
```
